subworkflows/local/eggnog.nf

```diff
@@ -0,0 +1,39 @@
+//
+// Run eggnog-mapper on called ORFs, first optionally downloading the required databases
+//
+
+include { EGGNOG_DOWNLOAD } from '../../modules/local/eggnog/download'
+include { EGGNOG_MAPPER   } from '../../modules/local/eggnog/mapper'
```

Comment:
Aren't there official modules yet? Should we contribute? Isn't anyone else using eggnog-mapper?

Reply:
Now that the module is working we could contribute by adding the module to nf-core. I can do it

Reply:
It is a bit tricky because of the download of a very large database. I know that James Fellow Yates have been working on other modules that require database download, so we should check what they look like. Phyloplan is one of them, I believe.